modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -520,6 +522,7 @@ action.VIEW_COMMERCE_PRODUCT_ATTACHMENTS=View Attachments
 action.VIEW_COMMERCE_PRODUCT_IMAGES=View Images
 action.VIEW_COMMERCE_PRODUCT_RULES=View Rules
 action.VIEW_COMMERCE_PRODUCT_TAX_CATEGORIES=View Tax Categories
+action.VIEW_COMMERCE_TERM_ENTRY=View Term Entry
```

Comment:
why?

Reply:
Because it's not translated and I see it on my tests

Reply:
please tell this to the QA